osmaxx-py/osmaxx/excerptexport/models/extraction_order.py

```diff
@@ -64,3 +88,8 @@ def extraction_configuration(self, value):
     @property
     def extraction_formats(self):
         return json.loads(self.extraction_configuration).get('gis_formats', None)
+
+    def set_status_from_conversion_progress(self, job_overall_progress):
+        # Don't update if we're already done or have errored out:
```

Comment:
Is this comment needed? To me, what it says is mostly evident from the code itself.